File: Allamvizsga/Allamvizsga/Scripts/Services/OwnerModel.js
function OwnerModel(data) {
    this.phoneNumber = ko.observable("");
    this.firstName = ko.observable("");
    this.lastName = ko.observable(null);
    this.email = ko.observable(null);
    if (data != null) {
        this.phoneNumber(data.PhoneNumber);
        this.email(data.Email);
        this.firstName(data.FirsName);
        this.lastName(data.LastName);
    
    }

}

File: Allamvizsga/Allamvizsga/Models/OwnerModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace Allamvizsga.Models
{
    public class OwnerModel
    {
        [Key]
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
        public string LastName { get; set; }
        public string FirstName { get; set; }
        public virtual ICollection<ServiceModel> Services { get; set; }

       
    }
}